task_reactredux/task1/src/containers/MenuContainer.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Switch, Route, Redirect, BrowserRouter, withRouter } from 'react-router-dom';
+import About from '../views/About/index';
+import CountersParentContainer from './CountersParentContainer';
+import NotFound from '../views/NotFound/index';
+import Menu from '../views/Menu/index';
+
+class MenuContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: false,
```

Comment:
Don't think that it could be boolean, as soon as Menu view shouldn't accept this as boolean